cmake/modules/SwiftUtils.cmake

```diff
@@ -152,6 +152,43 @@ function(swift_create_post_build_symlink target)
     COMMENT "${CS_COMMENT}")
 endfunction()
 
+function(swift_create_early_driver_symlinks target)
+  # Is there a better way to get the top-level `build` dir prefix?
+  set(driver_bin_dir "${CMAKE_CURRENT_BINARY_DIR}/../../../earlyswiftdriver-${SWIFT_HOST_VARIANT}-${SWIFT_HOST_VARIANT_ARCH}/release/bin")
+  set(swift_bin_dir "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+  # If early swift-driver wasn't built, nothing to do here.
+  if(NOT EXISTS "${driver_bin_dir}/swift-driver" OR NOT EXISTS "${driver_bin_dir}/swift-help")
+      return()
+  endif()
+
+  if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows")
+      set(cmake_symlink_option "copy_if_different")
+  else()
+      set(cmake_symlink_option "create_symlink")
+  endif()
+  set(cmake_symlink_option "create_symlink")
+
+  message("Creating early SwiftDriver symlinks.")
+  message("From: ${driver_bin_dir}/swift-driver")
+  message("To: ${swift_bin_dir}/swift-driver")
+  add_custom_command(TARGET "${target}" POST_BUILD
+  COMMAND
+    "${CMAKE_COMMAND}" "-E" "${cmake_symlink_option}"
+    "${driver_bin_dir}/swift-driver"
+    "${swift_bin_dir}/swift-driver"
+  COMMENT "${CS_COMMENT}")
+
+  message("From: ${driver_bin_dir}/swift-help")
+  message("To: ${swift_bin_dir}/swift-help")
+  add_custom_command(TARGET "${target}" POST_BUILD
```

Comment:
I think we have some cmake wrappers for creating symlinks. Can you git grep around for that.

Reply:
Replaced with the existing `swift_create_post_build_symlink`. 👍